thirdparty/jaspr_bootstrap/example/lib/pages/table_page.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:jaspr_bootstrap/components.dart';
+import 'package:jaspr/jaspr.dart';
+
+final List<String> headers = ["header 1", "header 2", "header 3", "header 4"];
+final List<List<String>> bodyData = [
+  ['1', '2', '3', '4'],
+  ['5', '6', '7', '8'],
+  ['9', '10', '11', '12'],
+  ['13', '14', '16', '32'],
+];
+
+class TablePage extends StatelessComponent {
+  @override
+  Iterable<Component> build(BuildContext context) sync* {
+    yield Table(
+      head: TableRow(
+        cells: headers
+            .map((e) => TableCell(
+                  text: e,
+                  cell: Cell.th,
+                ))
+            .toList(),
+      ),
+      body: bodyData
+          .map((e) => TableRow(
+                cells: e
+                    .map((e) => TableCell(
+                          text: e,
+                          cell: Cell.td,
```

Comment:
I don't like this `Cell.td` or `Cell.th`. Instead of this enum I would like to create classes `TableRow` and `TableCell` for better readability.. So one this example https://getbootstrap.com/docs/5.2/content/tables/ can look like:
```
Table(
  isDark: true,
  caption: "List of users", 
  header: TableHeader(
    row: TableRow(
      isBold: true,
      cells: [
        TableCell("#"), 
        TableCell("First"), 
        TableCell("Last"), 
        TableCell("Handle"), 
      ],
    ),
  ),
  body: TableBody(
    rows: [
      TableRow(
        cells: [
          TableCell("1", isBold: true), 
          TableCell("Mark"), 
          TableCell("otto"), 
          TableCell("@mdo"), 
        ],
      ),
      TableRow(
        cells: [
          TableCell("2", isBold: true), 
          TableCell("Jacob"), 
          TableCell("Thornton"), 
          TableCell("@fat"), 
        ],
      ),
      TableRow(
        cells: [
          TableCell("3", isBold: true), 
          TableCell("Larry the Bird", columns: 2), 
          TableCell("@twitter"), 
        ],
      ),
    ]
  ),
  footer: TableFooter(
    row: TableRow(
      cells: [
        TableCell("Footer"), 
        TableCell("Footer"), 
        TableCell("Footer"), 
        TableCell("Footer"), 
      ],
    ),
  ),
)

```


And your example can look like:
```


final List<String> headers = ["header 1", "header 2", "header 3", "header 4"];
final List<List<String>> bodyData = [
  ['1', '2', '3', '4'],
  ['5', '6', '7', '8'],
  ['9', '10', '11', '12'],
  ['13', '14', '16', '32'],
];


Table(
  header: TableHeader(
    row: TableRow(
      isBold: true,
      cells: [ 
        for (var header in headers) 
          TableCell(header),
      ],
    ),
  ),
  body: TableBody(
    rows: [
      for (var row in bodyData)
        TableRow(
          cells: [ 
            for (var cell in row) 
              TableCell(cell), 
          ],
        ),
    ]
  ),
)
```

Required will be only `body`. `header`, `footer` and `caption` can be optional.